About.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+
+    <header>
+        <div id="black-header " class="display_desktop"> 
+            <ul class="menu-black">
+               
+                <li><img src="images/facebook.png"></li>
+                <li><img src="images/twitter.png"></li>
+                <li>Logout</li>
+                <li>Mypage</li>
+                <li>Korean</li>
+            </ul>
+            </div>
+        <nav class="display_desktop navigation">
+            
+            <ul class="desktop_menu">
+                <li> <img src="images/logo.png" class="logo"></li>
+                <li>Program</li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li ><div id="CCampgain">CC Campaign</div></li>
+            </ul>
+            
+        </nav>
+        
+    </header>
+
+
+
+
```

Comment:
- [ ] [Optional] I noticed that there are many places in your code with extra spaces. 🚫 It's crucial to maintain good coding practices by avoiding excessive whitespace. Please take the time to update your code and clean up all the extra spaces in all the files. 📝 This is important for keeping the codebase neat and organized. Thank you! 🙏